src/app/templates/movie-detail.tpl

```diff
@@ -1,77 +1,50 @@
 <%
-if(typeof health === "undefined"){ health = false; };
 if(typeof synopsis === "undefined"){ synopsis = "Synopsis not available."; }; 
 if(typeof runtime === "undefined"){ runtime = "N/A"; };
-if (genre) {
-    for(var i = 0; i < genre.length; i++) {
-        genre[i] = i18n.__(genre[i]); 
-    }
+if (genres) {
```

Comment:
typeof should be used instead of if(), because if() will throw an error where typeof wont